apps/redi-talent-pool/src/components/organisms/ExpiredJobListings.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  TpJobListingStatus,
+  useTpJobListingPatchMutation,
+} from '@talent-connect/data-access'
+import { Button } from '@talent-connect/shared-atomic-design-components'
+import { formatDistance } from 'date-fns'
+import { Columns, Element } from 'react-bulma-components'
+import { useQueryClient } from 'react-query'
+import { EmptySectionPlaceholder } from '../molecules/EmptySectionPlaceholder'
+import JobPlaceholderCardUrl from './company-profile-editables/job-placeholder-card.svg'
+import { JobListingCard } from './JobListingCard'
+
+export function ExpiredJobListings({ jobListings }) {
+  const updateMutation = useTpJobListingPatchMutation()
+  const queryClient = useQueryClient()
+
+  const onReactivateCTAClick = (joblistingId) => {
+    updateMutation.mutate(
+      {
+        input: {
+          id: joblistingId,
+          status: TpJobListingStatus.Active,
+          expiresAt: new Date().setMonth(new Date().getMonth() + 6),
```

Comment:
In an ideal world, we wouldn't rely on the user agent's date. Then again we'd say the same thing about `status`. This should rather be located in a use case class on the server side. In the interest of time though, I find this entirely good enough. But maybe we can add a TODO "consider paying down this tech debt" note.

Continuing along the same train of thought, the same idea applies to various bits and pieces of code here and there... we do too much on the client-side and should rely more on the server-side. Not to mention payload validation. If only we had the resources :)